test/e2e/framework.go

```diff
@@ -548,6 +623,85 @@ func applyYAML(filename string, ns string) error {
 		return err
```

Comment:
it didn't log the error message before return

Reply:
ok